Codo-Baggins/game_crave
# Feature/game details

## Is this a fix or feature?
- Feature

## What is the change?
- Creates the ability for details of a searched game to be rendered to the page
- Allows a user to add or remove a game from their wish list
- Adds initial stylings to the page

## Where should the reviewer start?
-Line 27 of App.js and then lines 10-38 of GameDetails.js
## How should this be tested?
- Open the application in the browser
- Type the name of a video game into the input field
- Click the Search for game button
- Observe the game details rendering to the page
- Click the button at the bottom of the page to add or remove the game from the wish list
